lib/modules/manager/gitlabci-include/extract.ts

```diff
@@ -33,6 +42,57 @@ function extractDepFromIncludeFile(
   return dep;
 }
 
+function extractDepFromIncludeComponent(
+  includeComponent: GitlabIncludeComponent,
+  endpoint: string | undefined,
+): PackageDependency | null {
+  const componentReferenceMatch = componentReferenceRegex.exec(
+    includeComponent.component,
+  );
+  if (!componentReferenceMatch?.groups) {
```

Comment:
```suggestion
  const componentReferenceMatch = componentReferenceRegex.exec(
    includeComponent.component,
  )?.groups;
  if (!componentReferenceMatch) {
```

makes code simpler. needs some more changes below of cause